functions/tips/main.py

```diff
@@ -90,11 +90,15 @@ def scope_demo(request):
 
 # [START functions_tips_lazy_globals]
 # [START run_tips_global_lazy]
+from functools import lru_cache
+
 # Always initialized (at cold-start)
 non_lazy_global = file_wide_computation()
 
-# Declared at cold-start, but only initialized if/when the function executes
-lazy_global = None
+# Only initialized if/when the function executes
+@lru_cache()
+def lazy_computation():
+    return file_wide_computation()
 
 
 def lazy_globals(request):
```

Comment:
I think the name of the function doesn't reflect what it is once we merge this PR.

How about `global_and_cached_value` or something?